README.md

```diff
@@ -68,6 +68,15 @@ On Arch Linux you can install `hub` from official repository:
 $ sudo pacman -S hub
 ```
 
+#### FreeBSD
+
+On FreeBSD you can install a prebuilt `hub` package with
+[pkg(8)](http://man.freebsd.org/pkg/8):
+
+```console
+# pkg install hub
```

Comment:
Why does the command start with `#`?

Reply:
Well, sudo is not a part of the FreeBSD base system and pkg has to be run as root. I've used `#` instead of `$` to show that root access is required. :smiley: 

Cheers!